.github/trusted-contribution.yml

```diff
@@ -18,6 +18,7 @@ annotations:
 trustedContributors:
   - release-please[bot]
   - renovate[bot]
```

Comment:
Is this line still relavant? Should we remove it?

Reply:
I think it's fine to keep them just in case -- to cover any potential Renovate names